tools/build.py

```diff
@@ -3,21 +3,21 @@
 import os
 import sys
 import subprocess
+import shutil
 
 ######## GLOBALS #########
 MAINPREFIX = "z"
 PREFIX = "ace_"
 ##########################
 
 def tryHemttBuild(projectpath):
-    hemttExe = os.path.join(projectpath, "hemtt.exe")
-    if os.path.isfile(hemttExe):
+    if shutil.which("hemtt"):
         os.chdir(projectpath)
-        ret = subprocess.call([hemttExe, "pack"], stderr=subprocess.STDOUT)
-        print("Using hemtt: {}".format(ret));
+        ret = subprocess.call(["hemtt", "pack"], stderr=subprocess.STDOUT)
```

Comment:
What does `hemtt pack` do?

Reply:
oh, that's a 0.7.6 thing
we can probably just remove this script
the mikro part is probably broken and the hemtt stuff is outdated

Reply:
Could also replace a bunch of the python scripts with hemtt scripts

I think this PR is fine as is, and a follow up PR to remove what doesn't work / is outdated, and add HEMTT scripts equivalents (or commands in HEMTT if needed)